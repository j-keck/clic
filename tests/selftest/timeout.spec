# clic --cmdline 'sleep 1' --timeout 1ms
DUMMY
