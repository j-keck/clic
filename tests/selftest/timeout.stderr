Failure: Timeout waiting for response
